apps/portal/src/content/docs/components/commit-copy-action.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Commit Copy Action
+description: Commit Copy Action component documentation
+---
+
+The Commit Copy Action component provides a UI element for displaying a commit SHA and copying it to the clipboard. It also allows navigation to the commit details page.
+
+import { DocsPage } from "@/components/docs-page";
+
+<DocsPage.ComponentExample
+  client:only
+  code={`
+  <div className="flex">
+    <CommitCopyActions sha="abc1234" toCommitDetails={({ sha }) => "/commit/" + sha} />
+  </div>`}
```

Comment:
```suggestion
  code={`<div className="flex">
    <CommitCopyActions sha="abc1234" toCommitDetails={({ sha }) => "/commit/" + sha} />
</div>`}
```

Reply:
Not something to fix in this PR, but the layout of the component should be baked into the component. Must revisit this at some point and bring the flex layout into the component itself.